bulk/bulk.go

```diff
@@ -0,0 +1,47 @@
+package bulk
+
+import (
+	"errors"
+
+	"github.com/g8rswimmer/go-sfdc/session"
+)
+
+const bulk2Endpoint = "/jobs/ingest"
+
+// Resource is the structure that can be just to create bulk 2.0 jobs.
```

Comment:
This comment is not very intuitive in my opinion.  Feel free to tell me to pound sand.

Reply:
I really don't know what to say in the description.  It is mainly a resource for creating bulk 2.0 jobs.